### PR TITLE
#84 Update UI styling for Profile Page

### LINKED ISSUE #84: UI Styling of Profile Page

### PR DESCRIPTION
**Pull request by EvelynL09**
resolves #84

### CODE REVIEW

File: frontend/src/components/UserProfiles/ProfileListings.js [hunk 1]
```diff
@@ -18,23 +20,27 @@ const ProfileListings = ({ rides, handleError }) => {
...
     const activeRideList = activeRides.map((ride, index) => (
-        <RideProfile
-            key={index}
-            ride={ride}
-            isActive
-            handleError={handleError}
-            setWaypoints={setWaypoints}
-            setLineString={setLineString}
-        />
+        <List.Item className='ride-list-item'>
```
Comment (micheyale) [inline frontend/src/components/UserProfiles/ProfileListings.js:23]:
```suggestion
        <List.Item className="ride-list-item">
```

File: frontend/src/components/UserProfiles/ProfileListings.js [hunk 2]
```diff
@@ -18,23 +20,27 @@ const ProfileListings = ({ rides, handleError }) => {
...
     ));
     const completedRideList = completedRides.map((ride, index) => (
-        <RideProfile
-            key={index}
-            ride={ride}
-            handleError={handleError}
-            setWaypoints={setWaypoints}
-            setLineString={setLineString}
-        />
+        <List.Item className='ride-list-item'>
```
Comment (micheyale) [inline frontend/src/components/UserProfiles/ProfileListings.js:35]:
```suggestion
        <List.Item className="ride-list-item">
```

File: frontend/src/components/UserProfiles/ProfileRequests.js [hunk 3]
```diff
@@ -44,12 +44,12 @@ class ProfileRequests extends Component {
...
 
         return(
-            <div>
+            <div style={{width: "70%", margin: "auto"}}>
                 {
                     confirmedCount > 0 &&
                     <div className='confirmedRides'>
                         <div>{confirmedCount} Confirmed {confirmedCount > 1 ? "Rides": "Ride"}</div>
-                        <List className='requestsList' divided animated>
+                        <List className='requestsList-profile' divided verticalAlign="middle">
```
Comment (micheyale) [inline frontend/src/components/UserProfiles/ProfileRequests.js:52]:
```suggestion
                        <List className="requestsList-profile" divided verticalAlign="middle">
```

File: frontend/src/components/UserProfiles/ProfileRequests.js [hunk 4]
```diff
@@ -66,7 +66,7 @@ class ProfileRequests extends Component {
                     pendingCount > 0 &&
                     <div className='pendingRequests'>
                         <div>{pendingCount} Pending {pendingCount > 1 ? "Requests": "Request"}</div>
-                        <List className='requestsList' divided animated>
+                        <List className='requestsList-profile' divided verticalAlign="middle">
```
Comment (micheyale) [inline frontend/src/components/UserProfiles/ProfileRequests.js:69]:
```suggestion
                        <List className="requestsList-profile" divided verticalAlign="middle">
```

**Review by micheyale — CHANGES_REQUESTED** [review]

**Review by micheyale — APPROVED** [review]

**Comment by micheyale** [comment]
@EvelynL09 All good? Anymore incoming changes?

File: frontend/src/components/Rides/RideProfile.js [hunk 5]
```diff
@@ -208,27 +208,32 @@ class RideProfile extends Component {
...
-                    </h6>
+            <div className="ride-profile">
+                <Header as="h5" className="ride-header">
+                    <span>
+                        {/* Click header to view ride */}
+                        <a className="ride-link" href={'/ride/' + driverID + '/' + _id}>
+                            <div className="large-bold-font">{startLoc.city}, {startLoc.state}
+                            {/* <br/> */}
+                            {/* <span className="lighter-font">Starting Location</span> */}
+                                <Icon id="arrow" size="big" name="long arrow alternate right" />
```
Comment (EvelynL09) [inline frontend/src/components/Rides/RideProfile.js:219]:
```suggestion
                                <Icon id="arrow" size="large" name="long arrow alternate right" />
```

**Review by micheyale — APPROVED** [review]

**Pull request merged**